creator-node/src/monitors/MonitoringQueue.js

```diff
@@ -71,29 +77,48 @@ class MonitoringQueue {
    * them on init
    */
   async seedInitialValues() {
-    await this.refresh(MONITORS.STORAGE_PATH_SIZE)
-    await this.refresh(MONITORS.STORAGE_PATH_USED)
+    await this.refresh(MONITORS.STORAGE_PATH_SIZE, 'STORAGE_PATH_SIZE')
+    await this.refresh(MONITORS.STORAGE_PATH_USED, 'STORAGE_PATH_USED')
   }
 
-  async refresh(monitor) {
-    const key = getMonitorRedisKey(monitor)
+  /**
+   * Refresh monitor in redis and prometheus (if integer)
+   * @param {Object} monitorVal Object containing the monitor props like { func, ttl, type, name }
+   * @param {*} monitorKey name of the monitor eg `THIRTY_DAY_ROLLING_SYNC_SUCCESS_COUNT`
+   */
+  async refresh(monitorVal, monitorKey) {
+    const key = getMonitorRedisKey(monitorVal)
     const ttlKey = `${key}:ttl`
 
     // If the value is fresh, exit early
     const isFresh = await redis.get(ttlKey)
     if (isFresh) return
 
-    const value = await monitor.func()
-    this.logStatus(`Computed value for ${monitor.name} ${value}`)
+    const value = await monitorVal.func()
+    this.logStatus(`Computed value for ${monitorVal.name} ${value}`)
+
+    // store integer monitors in prometheus
+    try {
+      if (PROMETHEUS_MONITORS.hasOwnProperty(monitorKey)) {
+        const metric = this.prometheusRegistry.getMetric(
+          this.prometheusRegistry.metricNames[`MONITOR_${monitorKey}`]
+        )
+        metric.set({}, value)
+      }
+    } catch (e) {
+      logger.debug(
```

Comment:
if anything above log 98 should be a debug and this one should be info/warn haha